packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/FlatTreeNode.ts

```diff
@@ -35,8 +36,8 @@ export function isPlaceholderNode(node: FlatTreeNode): node is PlaceholderNode {
 }
 
 /** @alpha */
-export function flattenNodes(rootNodes: PresentationTreeNode[]) {
-  return getFlatNodes(rootNodes, 1);
+export function useFlattenNodes(rootNodes: PresentationTreeNode[]) {
```

Comment:
Maybe rename to `useFlatTreeNodesList`? Also, I think it's about time we start incrementally adding API docs to the new future-public functions like this.

Reply:
changed to `useFlatTreeNodeList`